kimchi/src/circuits/polynomials/foreign_field_add/witness.rs

```diff
@@ -27,6 +27,18 @@ pub enum FFOps {
     Mul,
 }
 
+/// Implementation of the FFOps enum
+impl FFOps {
+    /// Returns the sign of the operation as a field element, or zero if it is a multiplication
+    pub fn sign<F: PrimeField>(&self) -> F {
+        match self {
+            FFOps::Add => F::one(),
+            FFOps::Sub => -F::one(),
+            FFOps::Mul => F::zero(),
```

Comment:
I don't think adding a Mul operation make sense, since both gates already have separate APIs and can't really be unified since they are structurally so different.  That is, for mul vs add, the operation type is implicit in the interface name you call.  However, I do think it makes sense to ffadd to have a `Sign` parameter or `FFOps` for add or subtract.

Reply:
I don't think it is worth having a Sign type, it is just internal use and used in just a couple of places. The sign kind of is inherent to the Add/Sub wording.

Reply:
I will remove Mul then.